recipes/agent.rb

```diff
@@ -5,15 +5,14 @@
 template 'zabbix_agentd.conf' do
   path node['zabbix']['agent']['config_file']
   source 'zabbix_agentd.conf.erb'
-  unless node['platform_family'] == 'windows'
+  case node['platform_family']
```

Comment:
Can we do what we did with [`_agent_common_directories.rb`](https://github.com/paxos-bankchain/zabbix/pull/5#discussion_r198286971) and move the conditional outside of the resource declaration?